source/class/qx/tool/cli/commands/Package.js

```diff
@@ -172,12 +163,6 @@ qx.Class.define("qx.tool.cli.commands.Package", {
           qx.tool.compiler.Console.info(`>>> Saved library data to ${lockfileModel.getRelativeDataPath()}`);
         }
       }
-      if (compileConfigModel.isDirty()) {
```

Comment:
Can this be safely deleted? Are all changes to compile.json manually safed?

Reply:
yes, because Package no longer uses it - but no, because Install didn't have the save (see next commit)